src/main/java/edu/ie3/datamodel/io/factory/timeseries/TimeSeriesMappingFactory.java

```diff
@@ -38,4 +38,9 @@ protected TimeSeriesMappingSource.MappingEntry buildModel(EntityData data) {
     UUID timeSeries = data.getUUID(TIME_SERIES);
```

Comment:
Could you finish the adaptation of `TimeSeriesMappingFactory` here to not use UUIDs? Also the new method below